lib/components.js

```diff
@@ -363,7 +363,9 @@ class DatePicker extends Component {
       'minimumDate',
       'minuteInterval',
       'mode',
-      'timeZoneOffsetInMinutes'
+      'timeZoneOffsetInMinutes',
+      'background',
```

Comment:
If you agree I would:

- move the `config` option from `DatePicker` to `Component` in order to provide this new useful feature to all factories
- reuse the `mode` option to set the picker type
- keep the `format` custom option inside `config` since is specific to this template implementation
- move the `background` custom option inside `config` since is specific to this template implementation